src/assets/scss/index.scss

```diff
@@ -1 +1,10 @@
 // Import your SCSS files here
+@import './vendor/index.scss';
```

Comment:
Gulp automatically imports SASS files inside vendor folder and minify it, so isn't needed import here too, remove this import.